spec.md

```diff
@@ -48,31 +48,55 @@ be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
 This specification defines the following terms:
 
-- *Platform*: The software that will manage the cloud environment into which
-  Applications and Service Brokers are provisioned.  Users will not directly
-  provision Services from Service Brokers, rather they will ask the Platform
-  (ie. their cloud provider) to manage Services and interact with the
-  Service Brokers for them.
-
-- *Service*: A managed software offering that can be used by an Application.
-  Typically, Services will expose some API that can be invoked to perform
-  some action. However, there can also be non-interactive Services that can
-  perform the desired actions without direct prompting from the Application.
-
-- *Service Broker*: Service Brokers manage the lifecycle of Services. Platforms
-  interact with Service Brokers to provision, and manage, Service Instances
-  and Service Bindings.
-
-- *Service Instance*: An instantiation of a Service offering.
-
-- *Service Binding*: The representation of an association between an
-  Application and a Service Instance. Often, Service Bindings, will
-  contain the credentials that the Application will use to communicate
-  with the Service Instance.
+- *Platform*: The software that exposes a user-facing catalog of Service
+  offerings, aggregated from those offered by all Service Brokers. The
+  platform is the API client for Service Brokers. Users will not directly
+  provision Services from Service Brokers; rather they will ask the Platform
+  (ie. their cloud provider) to manage Services for them. Typically the platform
+  also manages the cloud environment into which Applications are provisioned.  
+
+- *Service Broker*: Service Brokers implement the Service Broker API, providing
+  an abstraction to Platforms for management of lifecycle operations across all
```

Comment:
Not sure about the 'all services' language here.  "across the services they offer"